htdocs/cron/class/cronjob.class.php

```diff
@@ -1057,7 +1057,10 @@ public function run_jobs($userlogin)
 			if (!$error)
 			{
 				$result = $langs->load($this->module_name);
-				$result = $langs->load($this->module_name.'@'.$this->module_name); // If this->module_name was an existing language file, this will make nothing
+				if($result<0){
+					$result = $langs->reload($this->module_name.'@'.$this->module_name);
```

Comment:
I think we can do simplest.
What do you have for value juste after the first $langs->load($this->module_name); for
print $langs->_tab_loaded[$this->module_name] ?    (set _tab_loaded as public to be able to make the test)
Do you have 2 ?

Reply:
yes

Reply:
A good usage of modules langs files is to use filename@modulename with filename != modulename but in facts modules usually uses a same name for module and lang file.

Reply:
What is hurting me is introducing a new method reload that contains some similar logic than load.
Why not just adding a second parameter $forceloadifalreadynotfound :

$result = $langs->load($this->module_name);
$result = $langs->load($this->module_name.'@'.$this->module_name, 1);

With this parameter, we should be able to disable the test done on cache into the load method.

if (!empty($this->_tab_loaded[$newdomain]))	// File already loaded for this domain
with
if (!empty($this->_tab_loaded[$newdomain]) && (empty(forceloadifalreadynotfound) || $this->_tab_loaded[$newdomain] !=2)	 // File already loaded and found and not forced for this domain

This is just theory, i do not test it but it looks simpler that your PR, don't you think ?

Reply:
I'm agree.
In fact my first comment https://github.com/Dolibarr/dolibarr/pull/12213#issuecomment-688822795 i says a similar thing 
but we are in a fix, so, modify a method signature could generate warnings in PHP >= 7 for instances of Dolibarr version 11 currently in production.
even if I'm pretty sure no many developers extends the Translate class...

My goal was to fix the version 11 with my fix and after add another fix  to the develop version by adding a new param in load method like $forceloadifalreadynotfound and refactor the reload method by simply calling load method with the new param to avoid php warning.

But if you think changing  the "load" method signature is impact-less so let's go.

Reply:
The problem is that it is difficult to manage fix in a way in a version and fix into another way into another.
This create a lot of troubles with merging maintenance branch, it also make impact analysis more difficult when we compare code between version, ...
Here we change the signature by adding a property that has a default value. Impact is very minor and very easy to fix if someone has extended the load method. So yes, i prefer we take the risk by adding a new parameter, for this case at least.

Reply:
ok good for me, I will use the simplest fix